hawkeye-worker/Cargo.toml

```
@@ -1,20 +1,19 @@
 [package]
 name = "hawkeye-worker"
 version = "0.1.0"
-authors = ["Rafael Caricio <rafael@caricio.com>"]
+authors = ["Rafael Caricio <rafael@caricio.com>", "Lyle Scott <lyle@ls3.io>"]
 edition = "2018"
 license = "MIT"
 repository = "https://github.com/cbsinteractive/hawkeye"
 
-
 [dependencies]
 hawkeye-core = { path = "../hawkeye-core" }
 image = "0.23"
 glib = "0.14.8"
 gstreamer = "0.17.4"
 gstreamer-app = "0.17.2"
 derive_more = "0.99.17"
-dssim = "2.11"
+dssim = "3.1.0"
```

```suggestion
dssim = "2.11"
```

See comment on main PR about this change... and why it needed to be the old version.